src/Polly.Shared/PolicyResult.cs

```diff
@@ -103,12 +116,25 @@ internal PolicyResult(TResult result, OutcomeType outcome, Exception finalExcept
         /// </summary>
         public Context Context { get; }
 
-        internal static PolicyResult<TResult> Successful(TResult result, Context context)
+        /// <summary>
+        /// Builds a successful <see cref="PolicyResult" />
+        /// </summary>
+        /// <param name="result">The result</param>
+        /// <param name="context">The policy context</param>
+        /// <returns></returns>
+        public static PolicyResult<TResult> Successful(TResult result, Context context)
         {
             return new PolicyResult<TResult>(result, OutcomeType.Successful, null, null, context);
         }
 
-        internal static PolicyResult<TResult> Failure(Exception exception, ExceptionType exceptionType, Context context)
+        /// <summary>
+        /// Builds a failed <see cref="PolicyResult" />
+        /// </summary>
+        /// <param name="exception">The exception</param>
+        /// <param name="exceptionType">The exception type</param>
+        /// <param name="context">The policy context</param>
+        /// <returns></returns>
```

Comment:
((suggest add, for 'returns'))

`A <see cref="PolicyResult" /> representing a failed execution through the policy.`